checkpoint/runner.go

```diff
@@ -71,12 +71,7 @@ func checkpointDB(
 		if err != nil {
 			return nil, fmt.Errorf("atxs snapshot commitment: %w", err)
 		}
-		vrfNonce, err := atxs.VRFNonce(tx, catx.SmesherID, catx.Epoch+1)
-		if err != nil {
-			return nil, fmt.Errorf("atxs snapshot nonce: %w", err)
-		}
 		copy(atxSnapshot[i].CommitmentATX[:], commitmentAtx[:])
-		atxSnapshot[i].VRFNonce = vrfNonce
```

Comment:
This breaks checkpoint...

Reply:
This value is now obtained by `LatestN` function by just taking the value of the `nonce` column for the ATX. The value of `nonce` column is not the same as `VRFNonce` field stored in the blob, but rather the same nonce that is returned by `atxs.VRFNonce(tx, catx.SmesherID, catx.Epoch+1)` here